src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.util.datalog.DataLog;
+import edu.wpi.first.util.datalog.DoubleLogEntry;
+import edu.wpi.first.wpilibj.DataLogManager;
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.MotorController;
+import frc.robot.hardware.MotorController.MotorConfig;
+import frc.robot.classes.TunableNumber;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+
+public class BuddyBalanceSubsystem extends SubsystemBase {
+  public static double kBalancedPosition; // Buddy balance PID reference point when lifting a robot and engaging charge station
+  public static double kDeployedPosition; // Buddy balance PID reference point when setting down a robot/initial position when deployed
+  // TODO: Make the reference point constants and default motor PID values final when they are done being tuned with TunableNumbers
+  public static final double kDefaultMotorP = 1e-6;
+  public static final double kDefaultMotorI = 0;
+  public static final double kDefaultMotorD = 1e-6;
+  public static final int deployServoID = 15;
```

Comment:
Use a channel between 0-9. That's what is available on the roboRIO. Channel 15 maps to the MXP port, and that probably requires custom/different cabling.